lib/Mojolicious/Plugin/AssetPack/Pipe/Export.pm

```diff
@@ -0,0 +1,92 @@
+
+package Mojolicious::Plugin::AssetPack::Pipe::Export;
+
+use Mojo::Base 'Mojolicious::Plugin::AssetPack::Pipe';
+use Mojo::Util 'spurt';
+
+use File::Basename 'dirname';
+use File::Path 'make_path';
+use File::Spec;
+
+has 'export_dir';
+has 'use_checksum_subdir' => 1;
+
+sub process {
+	my ($self, $assets) = @_;
+
+	my $dir = $self->export_dir // $ENV{'MOJO_ASSETPACK_EXPORT_DIR'};
+	unless ( defined $dir and -e -w -d $dir ) {
```

Comment:
from `perldoc -f -x`:

> As of Perl 5.10.0, as a form of purely syntactic sugar, you can
            stack file test operators, in a way that "-f -w -x $file" is
            equivalent to "-x $file && -w _ && -f _". (This is only fancy
            syntax: if you use the return value of "-f $file" as an argument
            to another filetest operator, no special magic will happen.)